src/ExpressionEvaluator/CSharp/Test/ResultProvider/FullNameTests.cs

```diff
@@ -1001,7 +1001,7 @@ public void MangledName_SimplifyBackingField_02()
 
             IDkmClrFullNameProvider2 fullNameProvider = new CSharpFormatter();
             var inspectionContext = CreateDkmInspectionContext();
-            Assert.Equal("<StringParameter>P", fullNameProvider.GetClrNameForField(inspectionContext, new DkmClrRuntimeInstance(assembly).Modules[0], fieldToken));
+            Assert.Equal("StringParameter", fullNameProvider.GetClrNameForField(inspectionContext, new DkmClrRuntimeInstance(assembly).Modules[0], fieldToken));
```

Comment:
>Assert.Equal("StringParameter", fullNameProvider.GetClrNameForField(inspectionContext, new DkmClrRuntimeInstance(assembly).Modules[0], fieldToken)); [](http://example.com/codeflow?start=12&length=148)

This behavior change feels unexpected. The scenario doesn't involve captured primary constructor parameters